src/osd/ReplicatedPG.cc

```diff
@@ -10954,9 +10954,6 @@ int ReplicatedPG::recover_backfill(
   }
   backfill_info.trim_to(last_backfill_started);
 
-  hobject_t backfill_pos = MIN_HOBJ(backfill_info.begin,
```

Comment:
but this variable is still referenced at https://github.com/ceph/ceph/pull/9120/files#diff-72747d40a424e7b5404366b557ff12a3L11167